converters/fromZigbee.js

```diff
@@ -3745,6 +3745,21 @@ const converters = {
             ratelimitedDimmer(model, msg, publish, options, meta);
         },
     },
+    terncy_contact: {
+        cluster: 'genBinaryInput',
+        type: 'attributeReport',
+        convert: (model, msg, publish, options, meta) => {
+            return {contact: (msg.data['presentValue']==0)};
+        },
+    },
+    terncy_temperature_simple: {
```

Comment:
Why not use https://github.com/Koenkk/zigbee-herdsman-converters/pull/1209/files#diff-63bec0da09650ada978527cc004902e2R3763 ?

Reply:
**I tried that first, but it was not working.** The difference is that my log (according to your guide) did not show anything about `readResponse` type (and I do not know what it is used for) only for `attributeReport`. Do not know what the actual problem was, so I did not give a second thought and created a new one. Keep the request open if you want and I will give it a second look soon.

Reply:
Hello again. Indeed there was **no** problem with the already existed converter `terncy_temperature`. I mistook it for problem because it delays to send the temperature info for a lot of minutes (thus why no info on log were present). Having said that I removed the redundant converter `terncy_temperature_simple`. Thank you for pointing that out.